src/adapter/src/coord.rs

```diff
@@ -417,11 +420,19 @@ impl<S: Append + 'static> Coordinator<S> {
             }
         }
 
-        self.initialize_storage_read_policies(
-            persisted_source_ids,
-            DEFAULT_LOGICAL_COMPACTION_WINDOW_MS,
-        )
-        .await;
+        for (instance_id, collection_ids) in persisted_source_ids {
```

Comment:
This wasn't changed in this PR, but do you remember why these read policies are not just included in the call below on line 669 to `self.initialize_read_policies(policies_to_set, DEFAULT_LOGICAL_COMPACTION_WINDOW_MS)
.await;`? Was there a specific reason to keep these separate?

Reply:
Good question! I think this was just an oversight, but @lluki can correct me if I'm wrong. I pushed a second commit to change this.

Reply:
Good question indeed, it's probably an oversight, i remember changing this a couple of times and possibly a nonsensical version slipped in.